### PR TITLE
Feat/add markets table

### LINKED ISSUE #766: Markets: Add Markets Table
### Subject of the issue
The dashboard currently has a list of links in the left sidebar. We would like to activate the _Markets_ tab, which when clicked should show the `FuturesMarketsTable`. 

Screen Shot 2022-04-28 at 17 16 52

Create the Markets Overview page and add the Markets table to it. Also add the tabs (Futures Markets) so once #762 is done, we can switch here between Futures and Spot Markets.

### Rewards
25 SNX

### Deadline
May 4th, 2022

### PR DESCRIPTION
**Pull request by 0xwontonsoup**
## Description
Fixes #766 

## How Has This Been Tested?
1. I worked backward by copying the `Overview` page to a new page
2. Renamed the new page `Markets` 
3. Removed any unnecessary content 

## Screenshots (if appropriate):
Screen Shot 2022-04-29 at 12 33 45 PM

### CODE REVIEW

**Review by platschi — APPROVED** [review]
Works great and as expected!

File: queries/futures/useGetAverageFundingRateForMarkets.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ const useGetAverageFundingRateForMarkets = (
 	const { synthetixjs } = Connector.useContainer();
 	const futuresEndpoint = getFuturesEndpoint(network);
 
-	return useQueries<FundingRateResponse[]>(
+	return useQueries(
```
Comment (brossetti1) [inline queries/futures/useGetAverageFundingRateForMarkets.ts:37]: was this removed for a particular reason?

or should this be reverted?

**Review by brossetti1 — APPROVED** [review]
very nice small effecient commit - 💯 

left one comment but wont hold this up for that

File: queries/futures/useGetAverageFundingRateForMarkets.ts (hunk 1 above)
Comment (platschi) [reply queries/futures/useGetAverageFundingRateForMarkets.ts:37]: Lint was complaining about this

**Pull request merged**